### PR TITLE
Add no-password to step certificate p12

### LINKED ISSUE #414: Enable a flag to not require a password for p12 file
### What would you like to be added
In order to support certain automated functions for certificate packaging and then import into the Windows Cert Store,  `step certificate p12` should allow for no-password to be required.

Something like an `--insecure` or `--no-password` parameter option.

### PR DESCRIPTION
**Pull request by maraino**
### Description

This PR adds the flags `--no-password --insecure` to `step certificate p12` to encrypt a PKCS#12 archive with a blank password.

Fixes #414

### CODE REVIEW

File: command/certificate/p12.go [hunk 1]
```diff
@@ -96,26 +112,26 @@ func p12Action(ctx *cli.Context) error {
...
+				return err
+			}
 		}
-	}
 
-	if password == "" {
-		pass, err := ui.PromptPassword("Please enter a password to encrypt the .p12 file")
-		if err != nil {
-			return errors.Wrap(err, "error reading password")
+		if password == "" {
```
Comment (dopey) [inline command/certificate/p12.go:125]: Shouldn't we just fail here or accept that the user wanted to use an empty password. If the password file you selected was empty, it seems weird to then ask the user for another password.

**Review by dopey — APPROVED** [review]
I left a comment about a weird place in the logic. You don't have to change it if you don't want. Everything else looks good.

File: command/certificate/p12.go (hunk 1 above)
Comment (maraino) [reply command/certificate/p12.go:125]: I left the functionality that was already there. But I think it's better to use `--no-password --insecure` if you really want an empty string as the password

Comment (dopey) [reply command/certificate/p12.go:125]: Then maybe we should just fail if the password file is empty? 

We don't normally check that value. We just use it (I think).

Comment (maraino) [reply command/certificate/p12.go:125]: That also match how other commands act `step crypto jwk create`

**Pull request merged**